Snappy compression fails in Alpine OpenJDK Docker image

```
Caused by: java.lang.UnsatisfiedLinkError: /tmp/snappy-unknown-e9f8ee9f-93a7-4415-a437-ec9eaa38a22f-libsnappyjava.so: Error loading shared library ld-linux-x86-64.so.2: No such file or directory (needed by /tmp/snappy-unknown-e9f8ee9f-93a7-4415-a437-ec9eaa3
8a22f-libsnappyjava.so)
 Jun 14 18:13:21 ip-172-31-142-67 docker/ab73110cc978[868]: #011at java.lang.ClassLoader$NativeLibrary.load(Native Method)
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,8 +1,6 @@
-FROM registry.opensource.zalan.do/stups/openjdk:8-2-alpine
+FROM registry.opensource.zalan.do/stups/openjdk:8-26
 
 EXPOSE 8085
-
-RUN apk add snappy
 
 COPY target/zmon-scheduler-1.0-SNAPSHOT.jar /zmon-scheduler.jar
 COPY target/scm-source.json /

--- a/src/main/java/de/zalando/zmon/scheduler/ng/CeleryBody.java
+++ b/src/main/java/de/zalando/zmon/scheduler/ng/CeleryBody.java
@@ -13,17 +13,10 @@ import java.util.Map;
 public class CeleryBody {
 
     public String expires;
-    public boolean utc = true;
     public final List<Object> args = new ArrayList<>(); // Order: CeleryCommandArg, CeleryAlertArg*
-    public String chord = null;
-    public String callbacks = null;
-    public String errbacks = null;
-    public String taskset = null;
     public String id;
-    public int retries = 0;
     public String task = "check_and_notify";
     public final List<Long> timelimit = new ArrayList<>(2);
-    public String eta;
     public final Map<String, Object> kwargs = new HashMap<>();
 
     public static class CeleryCommandArg {

